kong/plugins/ratelimiting/access.lua

```diff
@@ -14,23 +15,44 @@ function _M.execute(conf)
   else
     identifier = ngx.var.remote_addr
   end
+  
 
-  -- Load current metric for configured period
-  local current_metric, err = dao.ratelimiting_metrics:find_one(ngx.ctx.api.id, identifier, current_timestamp, conf.period)
-  if err then
-    return responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
-  end
+  local least_remaining_limit = 99999999 -- A huge number so that any limit will be actually less than this
+
+  for _, period_conf in ipairs(conf.limit) do
+    local period, limit = unpack(stringy.split(period_conf, ":"))
+    
+    limit = tonumber(limit)
```

Comment:
`period_limit`, like in the code snippet. I find this to be very confusing and would like other contributors to quickly understand this code.